### ISSUE TITLE
Jobs in general

### ISSUE THREAD

**Issue by Error3000** [DeathByKorea/BrokeProtocol#387]
Basically I want to say that some jobs are better than others, the 2 new jobs are good for rp but they require a lot of work and they give much less money than other jobs.
     Firefighter is a good option if serwer is populated with terrorists but then you can get killed while doing your job so you lose all your money. You also dont get any weapon to defend yourself.
     You can also be a medic which I think is not a good option because morphine is expensive and some people like to attack medics just to get this 2-3 morphine. Medics dont have any weapon to defend themselfes.
     Gang members do have weapons but they are a good target for people who choose to be a police because they will always have some illegal stuff.
     Basically what I wanted to say with this wonderful english is that the police is the most op work for now. You get pistol, tazer and handcuffs, you spawn with them every time you die and you can easily get loads of money while you are still able to defend yourself. You can handcuff anyone except other police officers which is again op and with current mechanics you can by selling illegal things and doing other ,,police stuff'' easily get 15k in about 10min.  Again, sorry for any language related mistakes and I hope it will change in the future so police wont be the most op option in the game.
